core/types/block.go

```diff
@@ -33,25 +33,53 @@ import (
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
+var (
+	EmptyRootHash  = DeriveSha(Transactions{})
+	EmptyUncleHash = CalcUncleHash(nil)
+)
+
+var (
+	errMissingHeaderMixDigest = errors.New("missing mixHash in JSON block header")
+	errMissingHeaderFields    = errors.New("missing required JSON block header fields")
+	errBadNonceSize           = errors.New("invalid block nonce size, want 8 bytes")
+)
```

Comment:
Again, I understand you probably define them just so you don't have to write the same error text multiple times, but I think they would be even more useful if they were exported and you could compare like `if err == types.ErrBadNonceSize` for outside consumers?